packages/xmpt/src/runtime.ts

```
@@ -0,0 +1,200 @@
+      // Validate envelope
+      EnvelopeSchema.parse(envelope);
+
+      if (transportMode === 'agentmail') {
+        // Use real AgentMail API
+        return sendViaAgentMail(to, body, envelope);
+      } else {
+        // Local mode - just store locally
+        messages.push(envelope);
+        // Echo back for local testing, but only if it's TO our inbox
+        if (envelope.to === inbox) {
+          callbacks.forEach(cb => cb(envelope));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)